src/uu/fmt/src/fmt.rs

```diff
@@ -331,3 +209,127 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
 
     0
 }
+
+pub fn uu_app() -> App<'static, 'static> {
+    App::new(executable!())
+        .version(crate_version!())
+        .about(ABOUT)
+        .arg(
+            Arg::with_name(OPT_CROWN_MARGIN)
+                .short("c")
+                .long(OPT_CROWN_MARGIN)
+                .help(
+                    "First and second line of paragraph
+        may have different indentations, in which
+        case the first line's indentation is preserved,
+        and each subsequent line's indentation matches the second line.",
```

Comment:
Might be a nice opportunity to fix this indentation (also for this other arguments)